### DISCUSSION TITLE
Unofficial website

### DISCUSSION THREAD

**Discussion by zwsren435** [valinet/ExplorerPatcher#2512] [General]
Bing (and therefore also DuckDuckGo) are still showing explorerpatcher.com (technocare.app) in search results for "explorerpatcher".

Non-tech savvy users who see this application recommended in news articles may accidentally download from this site, which could cause reputation issues for this project if the website owners decide to inject malware.

I'm not sure what needs to be done to get it delisted, so I just wanted to let you know in case you didn't.

Additionally, because this project doesn't have an official website (that appears in Google/Bing search results), those non-tech-savvy users who don't know how to download from Github may back out and click on the second search result, which happens to be this unofficial website (at least when I checked).

**Comment by pyrates999** [comment]
if it's a scam website, report it like this video mentions how: https://www.youtube.com/watch?v=0fIUiv9-UFk
